portfolio/src/main/webapp/index.html

```diff
@@ -117,6 +117,8 @@ <h6 class="title"> Research Assistant Intern </h6>
         <input type="text" id="name" name="name"> <br><br>
         <label for="comment"> Comment: </label> <br>
         <textarea rows="5" cols="75" type="text" id="text" name="text"> </textarea> <br> <br>
+        <input type="hidden" id="lat" name="lat">
```

Comment:
Cool, didn't know about `<input type="hidden">`

Reply:
Me neither! Lets you select and grab information from it in a way that <input type="text" id="lat" hidden> wouldn't which I found interesting and useful for my case.